zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/web/rest/ObjectManagementResource.kt

```diff
@@ -55,4 +61,11 @@ class ObjectManagementResource(
         objectManagementService.deleteById(id)
         return ResponseEntity.noContent().build()
     }
+
+    @GetMapping("/{configuration_id}/object")
+    fun getObjects(
+        @PathVariable configuration_id: UUID,
+        @PageableDefault pageable: Pageable
+    ): ResponseEntity<PageImpl<ObjectWrapper>> =
```

Comment:
Could you map the response to a Dto?

For the FE to be able to show the objects in a list, they need to have the response in a different format. 
See: https://ritense.atlassian.net/wiki/spaces/RIT/pages/1107296265/Object+management+-+API+specification#Get-object-list